src/components/hero-decorative/hero-decorative.js

```diff
@@ -1,6 +1,7 @@
 import './hero-decorative.scss'
 import template from './hero-decorative.pug'
-import heroData from '../../data/main-nav.json'
+import heroData from '../../data/data.json'
```

Comment:
`heroData` is probably not the best name since it will actually whole data from the whole page.